src/CreateRecipe/CreateRecipe.jsx

```diff
@@ -90,6 +95,21 @@ export default function CreateRecipe() {
             name="ingredients"
             required
           />
+          <Form.Group
+            controlId="formSelectMealType"
+            id="select-mealtype-form-group"
+            name="mealType"
+            onChange={onMealTypeChange}
+            required
+          >
+            <Form.Label>Select A Mealtype</Form.Label>
+            <Form.Control as="select">
```

Comment:
Not sure how we agreed on implementing the dropdown menus. In my option these options should come from the database to minimize duplicate code. The same options will be used for the feed preference. Something we should discuss.